src/main/java/dev/meyi/bn/utilities/EnchantedCraftingHandler.java

```diff
@@ -36,29 +39,31 @@
 
 
             String itemName = otherKeys.next();
-            double itemSellPrice = BazaarNotifier.bazaarDataRaw.getJSONObject(itemName).getJSONArray("sell_summary").getJSONObject(0).getDouble("pricePerUnit");
-            double itemBuyPrice = BazaarNotifier.bazaarDataRaw.getJSONObject(itemName).getJSONArray("buy_summary").getJSONObject(0).getDouble("pricePerUnit");
-            double ingredientPrice= 0d;
-            int ingredientCount;
-            double P= 0d;
-
-            for(int h = 0; h < BazaarNotifier.enchantCraftingList.getJSONObject("other").getJSONObject(itemName).getJSONArray("material").length(); h++){
-                if(h % 2 == 0){
-                    ingredientPrice = BazaarNotifier.bazaarDataRaw.getJSONObject(BazaarNotifier.enchantCraftingList.getJSONObject("other").getJSONObject(itemName).getJSONArray("material").getString(h)).getJSONArray("sell_summary").getJSONObject(0).getDouble("pricePerUnit");
-                }else{
-                    ingredientCount = BazaarNotifier.enchantCraftingList.getJSONObject("other").getJSONObject(itemName).getJSONArray("material").getInt(h);
-                    P += (ingredientPrice*ingredientCount);
-                }
-            }
-
+            try {  //sometimes the bazaarDataRaw sell_summary is empty

```

Comment:
Instead of using try-catch to see if it's empty, just check the JSONArray size for `sell_summary` and make sure it's greater than 0

(same for above)